verifier/utils.go

```diff
@@ -15,7 +17,11 @@ func getProjectFromAPIKey(c *client.RancherClient) (*client.Project, error) {
 func getServiceFromContainer(c *client.RancherClient, container *client.Container) (*client.Service, error) {
 	var svc *client.ServiceCollection
 	err := c.GetLink(container.Resource, "services", &svc)
-	if err != nil || len(svc.Data) == 0 {
+	if err != nil {
+		return nil, err
+	}
+	if len(svc.Data) == 0 {
+		err = fmt.Errorf("ERROR This container is not running inside a Rancher service")
```

Comment:
Its more idiomatic to do:
`import "errors"`
`return nil, errors.New("Container not running in a service")`

Since the Errorf isn't doing string substitution.